### DISCUSSION TITLE
Graphql Uploads in nextjs

### DISCUSSION THREAD

**Discussion by edw19** [MichalLytek/type-graphql#957] [Q&A]
on migration to typegraphql in noticed that file upload stopped working with next error

![image](https://user-images.githubusercontent.com/47750859/126817465-4f86e111-61c8-4c1d-be9d-71a08aab9ed9.png)

and  this config

![image](https://user-images.githubusercontent.com/47750859/126817537-fa8de34a-2406-4fdc-bad2-0501b67be32e.png)

Activating file upload in apollo server I have the following error

![image](https://user-images.githubusercontent.com/47750859/126817723-873b929d-7a91-40b1-b91c-95270fb873bd.png)

![image](https://user-images.githubusercontent.com/47750859/126822705-3cc58454-31d8-4c6a-ae92-6a87a7a66612.png)

the apollo client is config with apollo-client-upload

because it doesnot work

**Comment by MichalLytek** [comment]
See #37
